sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateSequencerPartitionContainerDao.java

```diff
@@ -199,6 +201,16 @@ public void restrictPaginationByPlatformType(Criteria criteria, PlatformType pla
     criteria.add(Restrictions.eq("model.platformType", platformType));
   }
 
+  @SuppressWarnings("deprecation")
+  @Override
+  public void restrictPaginationByKitName(Criteria criteria, String name, Consumer<String> errorHandler) {
+    criteria.createAlias("clusteringKit", "clusteringKit", CriteriaSpecification.LEFT_JOIN);
+    criteria.createAlias("multiplexingKit", "multiplexingKit", CriteriaSpecification.LEFT_JOIN);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "createAlias"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "LEFT_JOIN"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)